docker/al2023.Dockerfile

```diff
@@ -16,14 +16,20 @@ RUN pecl install igbinary && \
 
 ARG RELAY=v0.6.6
 
+# Install Relay dependencies
+RUN yum install -y \
+  http://download.opensuse.org/pub/opensuse/distribution/leap/15.5/repo/oss/x86_64/libopenssl1_1-1.1.1l-150500.15.4.x86_64.rpm \
+  http://download.opensuse.org/pub/opensuse/distribution/leap/15.5/repo/oss/x86_64/libhiredis1_1_0-1.1.0-bp155.1.6.x86_64.rpm \
+  http://download.opensuse.org/pub/opensuse/distribution/leap/15.5/repo/oss/x86_64/libck0-0.7.1-bp155.2.11.x86_64.rpm
```

Comment:
I'm just realizing, are the `download.opensuse.org` packages all available for `aarch64`?

Reply:
Yes, all three packages are available for `aarch64`

Reply:
In that case we need to add `ARCH=$(uname -m)` to all our Dockerfiles, so they build on ARM machines as well.

Are you able to add `download.opensuse.org` as a proper repository to the Dockerfile instead of direct links?

Reply:
Installing from repo is done but we need to discuss building on ARM machines

Reply:
https://docs.docker.com/build/ci/github-actions/multi-platform/